### PR TITLE
Add note about SPEC 0

### LINKED ISSUE #101: APE 18: Replace NEP 29 with SPEC 0
As proposed by @bnavigator here:

* https://github.com/astropy/astropy/pull/16376

For reference:

* https://numpy.org/neps/nep-0029-deprecation_policy.html (I don't see any deprecation notice here though)
* https://scientific-python.org/specs/spec-0000/

### PR DESCRIPTION
**Pull request by bnavigator**
Closes #101

### CODE REVIEW

File: APE18.rst [hunk 1]
```diff
@@ -5,12 +5,18 @@ author: Stuart Mumford and Thomas Robitaille
...
 
-date-last-revised: 2021 February 25
+date-last-revised: 2024 July 12
 
 type: Standard Track
 
 status: Accepted
 
+.. note::
+  NEP 29 has been superseded by the scientific python ecosystem coordination guideline
```
Comment (pllim) [inline APE18.rst:15]:
```suggestion
  NEP 29 has been superseded by the Scientific Python ecosystem coordination guideline
```

**Review by pllim — COMMENTED** [review]
I guess this works too. Thanks!

**Comment by pllim** [comment]
For completeness, I have mentioned this PR in astropy-dev mailing list and on the `#project` channel in Astropy Slack. Thanks again!

File: APE18.rst (hunk 1 above)
Comment (bnavigator) [reply APE18.rst:15]: I directly copied from NEP 29, where it is all lowercase. In SPEC 0 itself it is uppercase.

**Review by pllim — APPROVED** [review]
I have brought this up to CoCo multiple times and also no objection from astropy-dev and Slack. Let's merge. Thanks!

**Pull request merged**